components/wb-bubble/bubble.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * @title WET-BOEW Bubble plugin container
+ * @overview Plugin used to open a overlay in the middle of the page
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @duboisp @cTopCanada @GormFrank @Ricokola
+ */
+( function( $, window, wb ) {
+"use strict";
+
+/*
+ * Variable and function definitions.
+ * These are global to the plugin - meaning that they will be initialized once per page,
+ * not once per instance of plugin on the page. So, this is a good place to define
+ * variables that are common to all instances of the plugin on a page.
+ */
+var componentName = "wb-bubble",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	isNotif,
+	$document = wb.doc,
+	i18nDict = {
+		en: {
+			skip: "Skip to",
+			closeNotif: "Close notification"
+		},
+		fr: {
+			skip: "Passer à",
+			closeNotif: "Fermer la notification"
+		}
+	},
```

Comment:
(Optional but required for stable) Make this customizable from the plugin setting if possible.